pipeline/unified/build-unsigned-release-packages.yaml

```diff
@@ -0,0 +1,45 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+# Note: this will become the version of the released packages!
+# must be parseable by node-semver, so month leading zero is omitted
+name: $(Year:yyyy).$(Date:Mdd).$(Rev:r)
+
+strategy:
+    matrix:
+        windows: { vmImage: 'windows-latest', platform: 'windows' }
+        mac: { vmImage: 'macOS-10.14', platform: 'mac' }
+        linux: { vmImage: 'ubuntu-16.04', platform: 'linux' }
+
+pool:
+    vmImage: $(vmImage)
+
+steps:
+    - template: ../install-node-prerequisites.yaml
+
+    - template: ../electron/download-electron-mirror.yaml
+
+    - template: ../electron/electron-e2e-test-interactive.yaml
+
+    - template: ../electron/electron-e2e-publish-results.yaml
+
+    - script: yarn build:electron:all --unified-version=$(Build.BuildNumber) --app-insights-instrumentation-key=$(AppInsightsInstrumentationKey)
```

Comment:
Is `Build.Buildnumber` what we want for version? I thought we set up `name` above for this purpose

Reply:
I think they are identical - [here](https://docs.microsoft.com/en-us/azure/devops/pipelines/process/run-number?view=azure-devops&tabs=yaml) it says `name` is used to specify run number and that you can reference it via `Build.BuildNumber`. It looks like we can do [either](https://docs.microsoft.com/en-us/azure/devops/pipelines/process/run-number?view=azure-devops&tabs=yaml#how-can-you-reference-the-run-number-variable-within-a-script), so let me know if you think we should prefer one.